# This file configures the robots that spawn on the simulation and sets the position of each one.
# The configuration values should follow the next pattern:

# name_of_robot:
#    x:
#    y:
#    z:
#    yaw:

# Take into account that the values for the x, y and z fields depend on the simulation environment.
andino1:
  x: -1.158
  y: 2.0
  z: 0.1
  yaw: 0.0
andino2:
  x: 3.3
  y: 2.0
  z: 0.1
  yaw: 0.0
# andino3:
#   x: -1.
#   y: -0.94
#   z: 0.1
#   yaw: 0.0
# andino4:
#   x: 3.3
#   y: -0.75
#   z: 0.1
#   yaw: 0.0
